src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int BOUND = 100;
+    private Random number = new Random();
+    private ColorSupplier randomColor = new ColorSupplier();
+
+    public void setColor(ColorSupplier randomColor) {
+        this.randomColor = randomColor;
+    }
+
+    public ColorSupplier getColor() {
+        return randomColor;
+    }
```

Comment:
I don't think we need these methods

Reply:
We need. Because, the git checkout bot threw this problem all the time after I did instance randomColor with private modifier.